aidb/inference/bound_inference_service.py

```diff
@@ -189,63 +205,21 @@ async def infer(self, inputs: pd.DataFrame):
           results.append(df)
```

Comment:
Is it also possible to perform batch selection from the cache table? If so, we can create an issue to address this at a later time.

Reply:
It is possible. I'll handle it in different PR.